### ISSUE TITLE
Image request fired as soon as src is set

### ISSUE THREAD

**Issue by Josh-Cena** [mdn/content#26560]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/src

### What information was incorrect, unhelpful, or incomplete?

/

### What did you expect to see?

It's worth mentioning that a request to the specified URL is fired as soon as `src` is set, without even attaching the element to DOM.

### Do you have any supporting links, references, or citations?

<img width="471" alt="image" src="https://user-images.githubusercontent.com/55398995/235993744-184394dc-653e-4127-9ed3-54754dd5e7b2.png">

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/htmlimageelement/src`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/src
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/htmlimageelement/src/index.md
* Last commit: https://github.com/mdn/content/commit/a3d9f61a8990ba7b53bda9748d1f26a9e9810b18
* Document last modified: 2023-04-07T00:39:15.000Z

</details>

**Comment by hamishwillee** [comment]
Yes. Though note that the document must be fully active if the root node is a `DocumentFragment` (say created by `DOMParser.parseFromString`) the resource won't be downloaded. Also if loading is lazy it will be downloaded lazily.
